components/dashboard/src/projects/ConfigureProject.tsx

```diff
@@ -176,66 +157,39 @@ export default function () {
         });
     };
 
-    const buildProject = async () => {
+    useEffect(() => {
+        document.title = "Configure Project — Gitpod";
+    }, []);
+
+    const onNewWorkspace = async () => {
         if (!project) {
             return;
         }
-        setEditorMessage(null);
-        if (!!startPrebuildResult) {
-            setStartPrebuildResult(undefined);
-        }
-        if (!!prebuildInstance) {
-            setPrebuildInstance(undefined);
-        }
-        try {
-            setPrebuildWasTriggered(true);
-            if (!isEditorDisabled) {
-                await getGitpodService().server.setProjectConfiguration(project.id, gitpodYml);
-            }
-            const result = await getGitpodService().server.triggerPrebuild(project.id, null);
-            setStartPrebuildResult(result);
-        } catch (error) {
-            setPrebuildWasTriggered(false);
-            setEditorMessage(
-                <EditorMessage
-                    type="warning"
-                    heading="Could not run prebuild."
-                    message={String(error).replace(/Error: Request \w+ failed with message: /, "")}
-                />,
-            );
-        }
-    };
 
-    const cancelPrebuild = async () => {
-        if (!project || !startPrebuildResult) {
-            return;
-        }
-        setPrebuildWasCancelled(true);
-        try {
-            await getGitpodService().server.cancelPrebuild(project.id, startPrebuildResult.prebuildId);
-        } catch (error) {
-            setEditorMessage(
-                <EditorMessage
-                    type="warning"
-                    heading="Could not cancel prebuild."
-                    message={String(error).replace(/Error: Request \w+ failed with message: /, "")}
-                />,
-            );
-        } finally {
-            setPrebuildWasCancelled(false);
-        }
-    };
+        setButtonNewWorkspaceEnabled(false);
 
-    const onInstanceUpdate = (instance: WorkspaceInstance) => {
-        setPrebuildInstance(instance);
-    };
+        if (!isEditorDisabled) {
+            try {
+                setProgressMessage("Saving project configuration...");
+                await getGitpodService().server.setProjectConfiguration(project.id, gitpodYml);
 
-    useEffect(() => {
-        document.title = "Configure Project — Gitpod";
-    }, []);
+                setProgressMessage("Starting a new prebuild...");
```

Comment:
In my testing, starting a prebuild was required to actually use the new config. It wouldn't get picked up after the save without the prebuild. However, this may be a red-herring. I need to dig into the config & prebuild flow a bit more to understand why this is, and if in fact this is the case.

Will investigate!

Reply:
I've attempted to remove the prebuild step but the saved configuration is not reflected in the workspace. I will continue to investigate but in the meantime, this PR is functionally complete and will land.

Reply:
Sounds good! Thanks for taking a closer look, @easyCZ!